app/src/main/java/de/ph1b/audiobook/features/bookOverview/EditBookTitleDialogController.kt

```diff
@@ -50,18 +54,14 @@ class EditBookTitleDialogFragment : DialogFragment() {
 
   companion object {
 
-    val TAG: String = EditBookTitleDialogFragment::class.java.simpleName
+    val TAG: String = EditBookTitleDialogController::class.java.simpleName
     private const val NI_PRESET_NAME = "niPresetName"
     private const val NI_BOOK_ID = "niBookId"
 
-    fun newInstance(book: Book): EditBookTitleDialogFragment {
-      val args = Bundle()
-      args.putString(NI_PRESET_NAME, book.name)
-      args.putUUID(NI_BOOK_ID, book.id)
-
-      val dialog = EditBookTitleDialogFragment()
-      dialog.arguments = args
-      return dialog
-    }
+    fun newInstance(book: Book): EditBookTitleDialogController =
```

Comment:
Use a constructor

Reply:
I'm sorry, but I don't know what you mean. This method is using a constructor? Are you saying we should remove the `newInstance` method?